scripts/rv_expression_association/debug_maf_distributions.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import hail as hl
+# from bokeh.io import show
+from bokeh.io.export import get_screenshot_as_png
+from cpg_utils.hail_batch import output_path, init_batch 
+
+init_batch()
+mt = hl.read_matrix_table('gs://cpg-tob-wgs-test/tob_wgs_vep/v1/vep105_GRCh38.mt')
+mt = hl.experimental.densify(mt)
```

Comment:
If you wanted, you could at least filter to the chromosome that your target interval is on, to speed densification up.

Reply:
ditto